src/site/_data/paths/payments.json

```
@@ -25,7 +25,9 @@
           "title": "Build a web-based payment app",
           "pathItems": [
             "web-based-payment-apps-overview",
-            "registering-a-web-based-payment-app"
+            "registering-a-web-based-payment-app",
+            "orchestrating-payment-transactions-with-a-service-worker",
+            "handling-optional-payment-information-with-a-service-worker"
```

```suggestion
            "orchestrating-payment-transactions",
            "handling-optional-payment-information"
```

To keep these URLs more concise and shareable, what if we drop `with-a-service-worker`? We can keep "…with a service worker" in the titles, we'll just drop it from the URLs. Reminder that you'll need to rename the directories if you accept this change.

---

Great suggestion. Fixed.